libmscore/layout.cpp

```diff
@@ -2402,6 +2402,9 @@ static void breakCrossMeasureBeams(Measure* measure)
             if (!stf->show() || (stf->isTabStaff(measure->tick()) && stf->staffType(measure->tick())->slashStyle()))
                   continue;
 
+            if (!fstSeg)
+                  continue;
```

Comment:
It would probably be better just to return before the loop even starts, this function doesn't make sense if there is no chords or rests in the next measure.

Reply:
I wasn't too sure about what all the rest of thsat loop does...